src/midi/midi_fluidsynth.cpp

```diff
@@ -465,6 +482,15 @@ bool MidiHandlerFluidsynth::Open([[maybe_unused]] const char *conf)
 		mixer_channel->SetLowPassFilter(FilterState::Off);
 	}
 
+	// Double the baseline PCM prebuffer because MIDI is demanding
+	const auto render_ahead_ms = MIXER_GetPreBufferMs() * 2;
```

Comment:
Like in the MT-32 file, this seems pretty arbitrary.

Reply:
Added justification.